prow/cmd/deck/main.go

```diff
@@ -759,6 +760,9 @@ func handleBadge(ja *jobs.JobAgent) http.HandlerFunc {
 		w.Header().Set("Content-Type", "image/svg+xml")
 
 		allJobs := ja.ProwJobs()
+		sort.Slice(allJobs, func(i, j int) bool {
+			return allJobs[i].Status.StartTime.Before(&allJobs[j].Status.StartTime)
+		})
 		_, _, svg := renderBadge(pickLatestJobs(allJobs, wantJobs))
```

Comment:
Please move the sorting into `pickLatestJobs`. Right now the function name and description are misleading: https://github.com/kubernetes/test-infra/blob/1c5f719f89e6cbbfb38b93582edc7fe3e3b81fcc/prow/cmd/deck/badge.go#L86-L87